### PR TITLE
#BE-2111 Added Monitoring Documentation Page

### PR DESCRIPTION
**Pull request by burakberkkeskin**

### CODE REVIEW

**Review by csonuryilmaz — CHANGES_REQUESTED** [review]

File: docs/self-hosted-appcircle/configure-server/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+---
+title: Monitoring
+description: Instructions on migrating from a multi-node single drive MinIO configuration to a single-node single drive MinIO configuration
```
[the remark this answers is not in the document]
Comment (burakberkkeskin) [reply docs/self-hosted-appcircle/configure-server/monitoring.md]: Changed description.

File: docs/self-hosted-appcircle/configure-server/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+Change directory to Appcircle server.
+
+```bash
+cd appcircle-server
+```
+
+Print the Keycloak authentication credentials.
+
+```bash
+cat projects/spacetech/global.yaml | grep -A 5 "keycloak"
```
[the remark this answers is not in the document]
Comment (burakberkkeskin) [reply docs/self-hosted-appcircle/configure-server/monitoring.md]: Implemented `yq` command.

File: docs/self-hosted-appcircle/configure-server/monitoring.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+The monitoring system is designed to provide visibility into the application logs, enabling you to troubleshoot issues, monitor performance, and gain insights into the application's behavior.
+
+The subsequent sections of this document will guide you through the process of accessing the Grafana web interface, navigating the log data.
+
+:::info
+To access and use the monitoring capabilities, you must be running Appcircle server version `3.15.0` or later.
+:::
+
+## Accessing to Grafana Web UI
```
Comment (csonuryilmaz) [inline docs/self-hosted-appcircle/configure-server/monitoring.md:24]: We have an important requirement for this section.

In order to view the container logs on the UI, Appcircle services should be `up` and running `healthy`. So the UI can help diagnose only service issues while the Appcircle server is running. For troubleshooting the boot problems, especially when the `check` command after `up` reports `unhealthy`, CLI tools should be used for accessing the Appcircle service logs.

We should add an explanation for this. (caution box)

___
Note: CLI tools will be worked on in another issue later in the future. For now, we're only mentioning that the UI might not help to detect boot problems.

File: docs/self-hosted-appcircle/configure-server/monitoring.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+If you wish to configure a maximum size limit for automatic log deletion on the Journald, you can modify the relevant configuration settings.
+
+:::info
+Modifying the Journald configuration requires elevated privileges with `sudo` permissions, as it involves altering system-level settings.
+:::
+
+Edit the Journald config file.
+
+```bash
+sudo vim /etc/systemd/journald.conf
```
Comment (csonuryilmaz) [inline docs/self-hosted-appcircle/configure-server/monitoring.md:108]: As a convention, we're using `vi` in sample commands. (not `vim`)

**Review by csonuryilmaz — CHANGES_REQUESTED** [review]

File: docs/self-hosted-appcircle/configure-server/monitoring.md (hunk 4 above)
Comment (burakberkkeskin) [reply docs/self-hosted-appcircle/configure-server/monitoring.md:108]: Converted.

**Pull request merged**